.github/workflows/proj.yml

```diff
@@ -0,0 +1,44 @@
+name: Close Pull Request
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  merge_job:
+    # this job only runs when a PR's merged
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      # to use a private action, you have to check out the repo
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Node.js v12
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
```

Comment:
I really need to say "just install the packages/code needed for this action" instead of installing the whole repo